install_script.sh.template

```diff
@@ -1922,7 +1931,9 @@ for current_service in "${services[@]}"; do
     You will have to do it manually using the following command:
 
     $start_instructions\033[0m\n\n"
-
+    # It is possible that services were started during the installation phase, for example on CentOS7 (systemd < v237)
+    # In that case, we should stop them
+    $stop_instructions || true
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏ should we check if it's running and then try and stop it if it is, rather than always trying to stop it?

Reply:
IMO maybe we should check if they are running before running any `zypper install` command. This way, it would not stop a running agent service